### PR TITLE
Adding AgentInfo and Inventory endpoints

### PR DESCRIPTION
**Pull request by droessmj**
Adds support for AgentInfo & Inventory search endpoints

### CODE REVIEW

**Comment by droessmj** [comment]
@alannix-lw @kiddinn

File: laceworksdk/api/__init__.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 from dotenv import load_dotenv
 import configparser
+from laceworksdk.api.v2.inventory import InventoryAPI
```
Comment (alannix-lw) [inline laceworksdk/api/__init__.py:10]: This shouldn't be required.
```suggestion
```

**Review by alannix-lw — CHANGES_REQUESTED** [review]
Shouldn't need to import InventoryAPI twice, but other than that it looks good, I'll add tests shortly.

File: laceworksdk/api/__init__.py (hunk 1 above)
Comment (droessmj) [reply laceworksdk/api/__init__.py:10]: good catch. This snuck into my commit...will remove.

**Pull request merged**